manual/manual/refman/Makefile

```diff
@@ -10,7 +10,7 @@ CAMLLATEX = $(SET_LD_PATH) \
 TEXQUOTE = $(OCAMLRUN) $(TOOLS)/texquote2
 TRANSF = $(SET_LD_PATH) $(OCAMLRUN) $(TOOLS)/transf
 
-
+VPATH = extensions
```

Comment:
A simpler approach (in that it does not use fancy features such as `VPATH`) would be to have an EXTENSION_FILES variable with the extensions, and then include `$(addsuffix extension/, $(EXTENSION_FILES))` in `FILES`.

Reply:
Done, but with addprefix :-)